### PR TITLE
test: fix ci failures on main branch

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

- Fixes k8s test failure on the simulator
- Fixes VPC testcase failures for `test_set_sourcenat.py`

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@DaanHoogland this is still wip. Tests are still failing on GH CI,
```
+---------------------------------------+---------+-------+--------------------+
|                 Test                  | Result  | Time  |     Test file      |
+=======================================+=========+=======+====================+
| test_03_create_vpc_with_specified_sou | Failure | 7.666 | test_set_sourcenat |
| rce_nat_ip_address                    |         |       |                    |
+---------------------------------------+---------+-------+--------------------+
| test_04_change_source_nat_ip_address_ | Failure | 8.824 | test_set_sourcenat |
| for_vpc                               |         |       |                    |
+---------------------------------------+---------+-------+--------------------+
```

**Review by harikrishna-patnala — COMMENTED** [review]
Code LGTM. Let's run the tests.

@blueorangutan package

**Review by harikrishna-patnala — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by shwstppr** [comment]
@blueorangutan test

File: test/integration/smoke/test_set_sourcenat.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def validate_source_nat(self, network=None, vpc=None, ip=None):
         elif vpc:
             list_pub_ip_addr_resp = list_publicIP(
                 self.apiclient,
-                vpc=vpc.id,
+                vpcid=vpc.id,
```
Comment (DaanHoogland) [inline test/integration/smoke/test_set_sourcenat.py:255]: good spot

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@shwstppr changes look good, but I am investigating why the tests were skipped. I think we can merge though.

**Pull request merged**